src/components/inquiry_artwork_brick/index.tsx

```diff
@@ -0,0 +1,110 @@
+import * as React from "react"
+import * as Relay from "react-relay"
+import styled from "styled-components"
+import colors from "../../assets/colors"
+import Icon from "../icon"
+import ArtworkMetadata from "./metadata"
+
+const ImageContainer = styled.div`
+  height: 350px; 
+  width: 350px;
+  overflow: hidden;
+`
+
+const ImageOuterContainer = styled.div`
+  display: table;
+  width: 100%;
+  height: 100%;
+`
+const ImageInnerContainer = styled.div`
+  display: table-cell;
+  vertical-align: middle;
+  width: 100%;
+  height: 100%;
+  text-align: center;
+`
+
+const Image = styled.img`
+  max-width: 100%;
+  max-height: 350px;
+  margin: auto;
+`
+
+const SelectedArtworkOverlay = styled.div`
+  background-color: ${colors.purpleRegular};
+  height: 350px;
+  width: 350px;
+  position: absolute;
+  z-index: 10;
+  top: 0;
+  left: 0;
+  opacity: 0.8;
+  text-align: center;
+  line-height: 350px;
+`
+
+export class ArtworkState {
```

Comment:
I assume the second paragraph? Let’s reboot it, why are you using `class` here instead of `interface`?